src/Html/Builder.php

```diff
@@ -46,6 +46,11 @@ class Builder
      */
     public Collection $collection;
 
+    /**
+     * @var Collection<int, array>
```

Comment:
I suggest we enforce this to be a collection of Row?

```suggestion
     * @var Collection<int, Row>
```